### ISSUE TITLE
Wrong tag being closed after <?php ?> tags used

### ISSUE THREAD

**Issue by houseoftech** [mrhanlon/less-than-slash#34]
When closing a tag after a php tag was used it adds a closing tag for the tag that was opened inside php tags.

````
<div>
	<ul>
		<li></li>
		<?php if ($first == 1):?>
			<li></li>
		<?php endif;?>
	
````

This would close an `</li>`.

````
<div>
	<ul>
		<li></li>
		<?php echo "<a></a>";?>
	</a>
````

Here it closed an `</a>`.

**Comment by MarcoThePoro** [comment]
This should be fixed by c8018e0, it just need to be published

**Comment by mrhanlon** [comment]
I published v0.13.0, which includes this fix.
